### PR TITLE
Update page title to match existing profile.

### PR DESCRIPTION
**Pull request by davidjoy**
Learner Profile | edX

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -1,9 +1,9 @@
 <!doctype html>
 <html lang="en-us">
   <head>
-      <title>Account | edX</title>
+      <title>Learner Profile | edX</title>
```
Comment (albemarle) [inline public/index.html]: Should this be i18n'ed?

Comment (davidjoy) [reply public/index.html]: :O  That's a very good point.  We could dynamically update it once we get the translations...

Comment (albemarle) [reply public/index.html]: If we do this, the string should include the | edX part, so that the translators can flip it for RTL.

Comment (abutterworth) [reply public/index.html]: If we're going to dynamically update it should we put the user's name or username in there?

Comment (davidjoy) [reply public/index.html]: @abutterworth Possibly - it doesn't do that currently, fwiw.

**Pull request merged**